modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ItemActionForm.tsx

```diff
@@ -19,7 +19,37 @@ import RequiredMark from '../../components/RequiredMark';
 import ValidationFeedback from '../../components/ValidationFeedback';
 import openDefaultFailureToast from '../../utils/openDefaultFailureToast';
 import openDefaultSuccessToast from '../../utils/openDefaultSuccessToast';
-import {IFDSAction} from '../Actions';
+import {IFDSAction, SECTIONS} from '../Actions';
+
+const ACTION_TYPE = {
+	ASYNC: 'async',
+	HEADLESS: 'headless',
+	LINK: 'link',
+	MODAL: 'modal',
+	SIDEPANEL: 'sidePanel',
+};
+
+const ITEM_ACTION_TYPES = [
+	{
+		label: Liferay.Language.get('link'),
+		value: ACTION_TYPE.LINK,
+	},
+];
+
+const CREATION_ACTION_TYPES = [
+	{
```

Comment:
Could create a mechanism to filter action types (link, modal, sidepanel) for each action type (item, creation). Leave it as separate collections now.

Reply:
makes sense, it's fine imho

Reply:
@chestofwonders this may clash with your ongoing work, please talk to @juanjofgliferay